src/SpatialNavigation.ts

```diff
@@ -1039,12 +1071,12 @@ class SpatialNavigationService {
               ? siblingCutoffCoordinate >= currentCutoffCoordinate // vertical next
               : siblingCutoffCoordinate <= currentCutoffCoordinate // vertical previous
             : this.writingDirection === WritingDirection.LTR
-            ? isIncrementalDirection
-              ? siblingCutoffCoordinate >= currentCutoffCoordinate // horizontal LTR next
-              : siblingCutoffCoordinate <= currentCutoffCoordinate // horizontal LTR previous
-            : isIncrementalDirection
-            ? siblingCutoffCoordinate <= currentCutoffCoordinate // horizontal RTL next
-            : siblingCutoffCoordinate >= currentCutoffCoordinate; // horizontal RTL previous
+              ? isIncrementalDirection
+                ? siblingCutoffCoordinate >= currentCutoffCoordinate // horizontal LTR next
+                : siblingCutoffCoordinate <= currentCutoffCoordinate // horizontal LTR previous
+              : isIncrementalDirection
+                ? siblingCutoffCoordinate <= currentCutoffCoordinate // horizontal RTL next
+                : siblingCutoffCoordinate >= currentCutoffCoordinate; // horizontal RTL previous
```

Comment:
Please make sure to run Prettier on your code. All these small changes will be changed back next time we run Prettier. We will automate it with Husky in the future, but for now please only include changes that are relevant to your feature :)

Reply:
Hi @asgvard 
Thanks for your review :) I just reverted these unrelated changes. Could you take another quick look to see if everything is good to merge? 
Regarding adding Husky, I have used it before in another project and it is really convenient. Can I try and create another issue and PR to deal with it 😀 ?